privval/client_test.go

```diff
@@ -172,7 +222,12 @@ func TestSocketPVDeadline(t *testing.T) {
 	<-listenc
 }
 
-func TestRemoteSignerRetry(t *testing.T) {
+// TestRemoteSignerRetryTCPOnly will test connection retry attempts over TCP. We
+// don't need this for Unix sockets because the OS instantly knows the state of
+// both ends of the socket connection. This basically causes the
+// RemoteSigner.dialer() call inside RemoteSigner.connect() to return
+// successfully immediately, putting an instant stop to any retry attempts.
+func TestRemoteSignerRetryTCPOnly(t *testing.T) {
```

Comment:
On an unrelated note: This test seems misplaced in `client_test.go`. It doesn't really test any client related behaviour (ok the closing and reconnecting is sth the client might do but it is merely there to count the retries of the server). It basically spins off a server and doesn't test any client related code line.

Reply:
You're right - this looks more relevant to the `RemoteSigner` class. Would this be better placed in a new file `privval/remote_signer_test.go`?